lib/protocol/http/header/cache_control.rb

```diff
@@ -55,11 +59,27 @@ def no_store?
 					self.include?(NO_STORE)
 				end
 				
+				def must_revalidate?
+					self.include?(MUST_REVALIDATE)
+				end
+				
+				def proxy_revalidate?
+					self.include?(PROXY_REVALIDATE)
+				end
+				
 				def max_age
 					if value = self.find{|value| value.start_with?(MAX_AGE)}
 						_, age = value.split('=', 2)
 						
-						return Integer(age)
+						return Integer(age, exception: false)
```

Comment:
Is it okay to accept hexadecimal numbers here?

Reply:
The RFC specifies decimal only. Even though Integer() will parse hex if so prefixed, I don't think that's a concern in practice?

Reply:
I don't know about this field, but it can be a security issue to be more accepting of values, as someone can craft a request which has different interpretations to different software, and take advantage of that. Does the RFC define a strict format, e.g. a BNF? Or say anything?

Reply:
Yes, the BNF is the same for both [`max-age`](https://www.rfc-editor.org/rfc/rfc9111.html#section-5.2.2.1) and `s-maxage` and defined as [`delta-seconds`](https://www.rfc-editor.org/rfc/rfc9111.html#section-1.2.2) = `1*DIGIT`.

I'll tighten both up to accept only a decimal, positive integer.